File: Iniciante/1006 - Média 2/C/1006 - Média 2.c
#include <stdio.h>
 
int main() {
    double A, B, C, MED;
    
    scanf("%lf %lf %lf", &A, &B, &C); // recebe os valores 
    
	MED = (A*2+B*3+C*5)/10; // realiza o cálculo
    
    printf("MEDIA = %.1f\n", MED); // apresenta o resultado
    
    return 0;
}

File: Iniciante/1006 - Média 2/C#/1006 - Média 2.cs
using System;
using System.Globalization; // formata os valores para en-us (.)

class URI {
    static void Main() {
        double A, B, C, MED;

        A = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        B = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        C = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);

        MED = ((A*2) + (B*3) + (C*5)) / 10;

        Console.WriteLine("MEDIA = " + MED.ToString("0.0", System.Globalization.CultureInfo.InvariantCulture));
    }
}